ext/ffi_c/ClosurePool.h

```diff
@@ -36,6 +36,8 @@ struct Closure_ {
     void* info;      /* opaque handle for storing closure-instance specific data */
     void* function;  /* closure-instance specific function, called by custom trampoline */
     void* code;      /* The native trampoline code location */
+    void* pcl;
```

Comment:
What does the term `pcl` mean?

Reply:
The address for ffi_<p>rep_<c>losure_<l>oc.
Feel free to rewrite it :)